python/ray/air/BUILD

```diff
@@ -77,7 +77,7 @@ py_test(
     name = "test_data_batch_conversion",
     size = "small",
     srcs = ["tests/test_data_batch_conversion.py"],
-    tags = ["team:ml", "exclusive", "ray_data"],
+    tags = ["team:ml", "team:data", "exclusive", "ray_data"],
```

Comment:
for data team, can we also remove ml as owner?

Reply:
CC: @c21, @raulchen

Reply:
looking at test_data_batch_conversion.py, it looks to be testing mostly AIR utility methods, and not Ray Data directly. IMO we should not add this under data team?

Reply:
got you, I'll keep team:ml as the first oncall then, and just leave team:data there for now so that it's compatible with the ray civ2 interprets tests

I'll remove the team:data part once I migrate ml tests to civ2